tests/logout.test.js

```diff
@@ -0,0 +1,35 @@
+import LoginPage from '../pages/login.page';
+import LogoutPage from '../pages/logout.page';
+import HomePage from '../pages/home.page';
+import BasePage from '../pages/base.page';
```

Comment:
You don't need BasePage at this level, only for other pages to inherit from it at their specific pages files